UnitTests/TextViewTests.cs

```diff
@@ -1962,19 +1968,74 @@ public void WordWrap_WrapModel_Output ()
 			tv.Redraw (tv.Bounds);
 
 			string expected = @"
-This is 
-the 
-first 
-line.
-This is 
-the 
-second 
+This is
+the first
+ line.
```

Comment:
Per #1820, we all agree that TextView is not RENDERING word-wrapped text correctly. The correct rendering is in the deleted text here where the space was removed from the ` line.` line.

Right?

So, once the RENDERING behavior is fixed, this unit test will need to change.

I'd like you to rename these unit test methods to be clear on what is actually being tested.

In this case, the unit test is testing RENDERING.

There are (should be?) other unit tests that test that the INTERNAL REPRESENTATION of the `Text` is correct.

Reply:
The space is removed after set ReadOnly to true. But I'm seeing you didn't push all the code changes that I submitted to your pr branch. I'll submit a new pr to the gui-cs.

Reply:
> The space is removed after set ReadOnly to true. But I'm seeing you didn't push all the code changes that I submitted to your pr branch. I'll submit a new pr to the gui-cs.

You are not understanding me. 

The `ReadOnly` fix is a temporary HACK that will fix the way TextView *RENDERS* when WordWrap is enabled.

This DOES NOT FINALLY FIX #1820.  #1820 will ONLY be fixed correctly once we do a massive change to TextView that separates RENDERING of `Text` from how `Text` is stored internally. 

In THIS PR (#1867) I am requesting that you change the unit test NAMES to make it clear that what they are testing is just the RENDERING.

Reply:
@tig I hope have done what you asked. I'm a bit confused :-)